core/templates/dev/head/editor/feedback_tab/feedback_tab.html

```diff
@@ -159,3 +162,32 @@
     <button class="btn btn-default" ng-click="cancel()">Cancel</button>
   </div>
 </script>
+
+<script type="text/ng-template" id="modals/editorViewSuggestion">
+  <div class="modal-header">
+    <h3>Suggestion Review</h3>
+  </div>
+
+  <div class="modal-body">
+    <section style="width: 100%; height: 350px; padding: 10px;">
+      <div style="width: 49%; height: 100%; float: left;">
+        <strong>Current:</strong>
+        <div style="width:100%; height:100%; border:1px solid black; overflow: scroll;">
+          <p style="padding: 5px;"><[ oldContent ]></p>
+        </div>
+      </div>
+      <div style="width: 49%; height: 100%; float: right;">
+        <strong>Suggested:</strong>
+        <div style="width:100%; height:100%; border:1px solid black; overflow: scroll;">
+          <p style="padding: 5px;"><[ newContent ]></p>
```

Comment:
According to the design doc, this seems like it should be in a RTE, so the exploration creator can edit the suggestion before accepting it.